src/porepy/params/tensor.py

```diff
@@ -158,7 +245,10 @@ class FourthOrderTensor(object):
     """
 
     def __init__(
-        self, mu: np.ndarray, lmbda: np.ndarray, phi: Optional[np.ndarray] = None
+        self,
+        mu: np.ndarray,
+        lmbda: np.ndarray,
+        other_fields: dict[str, tuple[np.ndarray, np.ndarray]] = {},
```

Comment:
Don't initialize mutable optional arguments like that. See [here](https://web.archive.org/web/20200221224620id_/http://effbot.org/zone/default-values.htm) for an explanation.